upup/models/cloudup/resources/addons/digitalocean-cloud-controller.addons.k8s.io/k8s-1.8.yaml.template

```diff
@@ -52,12 +52,16 @@ spec:
         command:
           - "/bin/digitalocean-cloud-controller-manager"
           - "--cloud-provider=digitalocean"
-          - "--leader-elect=false"
+          - "--leader-elect=true"
```

Comment:
multiple masters is not supported yet but this enabling leader election now seemed pretty harmless